src/components/input/examples/Inputs.vue

```diff
@@ -377,29 +443,29 @@ export default {
   },
   watch: {
     $route(to) {
-      this.setBackground(to.query.background);
+      this.setBackground("primary");
     },
   },
   mounted() {
-    this.setBackground(this.$router.currentRoute.query.background);
+    this.setBackground("primary");
```

Comment:
Just to note I think eventually this will need to be switched back once we re-enable this background toggle feature

Reply:
Oh good catch. I think I set it to a single value because the warnings were cluttering my console. I'm going to set it back